### PR TITLE
added getSession api to fetch deviceUDID

### PR DESCRIPTION
**Pull request by saikrishna321**
## Change list

This PR helps users to fetch the sessionDetails.

https://github.com/appium/appium/issues/6550

https://github.com/appium/java-client/issues/215
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Received response 

```

  "status": 0,
  "value": {
    "platform": "LINUX",
    "webStorageEnabled": false,
    "takesScreenshot": true,
    "javascriptEnabled": true,
    "databaseEnabled": false,
    "networkConnectionEnabled": true,
    "locationContextEnabled": false,
    "warnings": {
      
    },
    "desired": {
      "app": "\/Users\/saikrisv\/git\/java-client\/java-client\/src\/test\/java\/io\/appium\/java_client\/IntentExample.apk",
      "platformName": "Android",
      "deviceName": "Android Emulator"
    },
    "app": "\/Users\/saikrisv\/git\/java-client\/java-client\/src\/test\/java\/io\/appium\/java_client\/IntentExample.apk",
    "platformName": "Android",
    "deviceName": "192.168.65.101:5555",
    "deviceUDID": "192.168.65.101:5555",
    "platformVersion": "5.0",
    "appPackage": "com.prgguru.android",
    "appWaitPackage": "com.prgguru.android",
    "appActivity": "com.prgguru.android.IntentExampleActivity",
    "appWaitActivity": "com.prgguru.android.IntentExampleActivity"
  },
  "sessionId": "0ac42e16-65c3-40ec-bcf0-fae7d9dd9a4a"
}
```

Sample Code:

java-client/src/test/java/io/appium/java_client/android/GetSessionDetailsTest.java

### CODE REVIEW

**Comment by saikrishna321** [comment]
@TikhomirovSergey @SrinivasanTarget Please review .

@email2vimalraj  your welcome to review this.

File: src/test/java/io/appium/java_client/android/GetSessionDetailsTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        if (driver != null) {
+            driver.quit();
+        }
+        if (service != null) {
+            service.stop();
+        }
+    }
+
+
+    @Test public void getDeviceUDIDTest() {
```
Comment (TikhomirovSergey) [inline src/test/java/io/appium/java_client/android/GetSessionDetailsTest.java]: Why did you add this test class? Maybe it makes sense to add this method to https://github.com/appium/java-client/blob/master/src/test/java/io/appium/java_client/android/AndroidDriverTest.java. What do you think?

**Comment by TikhomirovSergey** [comment]
I'm ok. But there is the remark.

**Comment by saikrishna321** [comment]
Yup, makes sense to move it AndroidDriverTest ..  will do that ....

**Comment by saikrishna321** [comment]
@TikhomirovSergey made changes as suggested .. Thoughts ?

**Comment by SrinivasanTarget** [comment]
@saikrishna321 I see AndroidDriverTest is broken. Can you add this line back https://github.com/appium/java-client/pull/403/files#diff-4cea3ff81e0a1323e7aeb653d2e4a366L30. Also squash your commits if possible.

**Comment by saikrishna321** [comment]
@SrinivasanTarget made the changes, sorry messed locally cant squash..

**Comment by email2vimalraj** [comment]
@saikrishna321 : I'm fine with the changes.

**Comment by TikhomirovSergey** [comment]
Ok. I'm merging it soon

**Comment by SrinivasanTarget** [comment]
Merged @saikrishna321
